sound/soc/intel/boards/Kconfig

```diff
@@ -611,11 +611,15 @@ config SND_SOC_INTEL_SOF_RT1308_MACH
 	depends on I2C && ACPI && GPIOLIB
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
 	select SND_SOC_RT1308
+	select SND_SOC_CS35L41_I2C
 	select SND_SOC_DMIC
+	select SND_SOC_HDAC_HDMI
+	select SND_SOC_INTEL_HDA_DSP_COMMON
 	select SND_SOC_INTEL_SOF_REALTEK_COMMON
+	select SND_SOC_INTEL_SOF_CIRRUS_COMMON
 	help
-	   This adds support for ASoC machine driver for Tigerlake platforms
-	   with RT1308 I2S audio codec.
+	   This adds support for ASoC machine driver for SOF platforms
+	   with RT1308/CS35L41 I2S audio codec.
```

Comment:
also need to update the line 609 and 610

Reply:
Change the config name to SND_SOC_INTEL_SOF_RT1308_CS35L41_MACH. Also update the Makefile.

Reply:
Would it be better to call it something like SND_SOC_INTEL_SOF_SSP_AMP_MACH and sof_ssp_amp.c?